soc/nxp/imx/imx9/Kconfig.soc

```diff
@@ -15,15 +15,15 @@ config SOC_MIMX9352_A55
 	help
 	  NXP i.MX93 A55
 
-config SOC_PART_NUMBER_MIMX9352CVUXK
+config SOC_PART_NUMBER_MIMX9352CVVXM
 	bool
 
 config SOC_PART_NUMBER_MIMX9352DVUXM
 	bool
 
 config SOC_PART_NUMBER
-	default "MIMX9352CVUXK" if SOC_PART_NUMBER_MIMX9352CVUXK
-	default "MIMX9352DVUXM" if SOC_PART_NUMBER_MIMX9352DVUXM
+	default "MIMX9352CVVXM" if SOC_PART_NUMBER_MIMX9352CVVXM
+	default "MIMX9352DVVXM" if SOC_PART_NUMBER_MIMX9352DVVXM
```

Comment:
Add:
```
default "MIMX9352XVVXM" if SOC_PART_NUMBER_MIMX9352XVVXM
default "MIMX9352AVTXM" if SOC_PART_NUMBER_MIMX9352AVTXM
```

Reply:
Just for my knowleadge, how to do choose which to add here?
Because, by looking [here](https://www.nxp.com/products/processors-and-microcontrollers/arm-processors/i-mx-applications-processors/i-mx-9-processors/i-mx-93-applications-processor-family-arm-cortex-a55-ml-acceleration-power-efficient-mpu:i.MX93), we have 6 part numbers for imx9352:
![image](https://github.com/user-attachments/assets/3e572af4-538d-44f5-b15d-0e9fe5632b49)

Reply:
was my question as well... the HAL side changed and the previous number we had was no longer part of the header file. I think @JiafeiPan indicated that the board and part is pre-production so things can and do change.